attack-theme/templates/general/base-template.html

```diff
@@ -91,18 +93,26 @@
             <div class="w-100 p-2 footer-responsive-break"></div>
             <div class="col">
                 <div class="footer-float-right-responsive">
-                    <div class="mb-1">
-                        <a href="https://twitter.com/MITREattack" class="btn btn-primary w-100">
-                            <!-- <i class="fa fa-twitter"></i> -->
-                            <img src="/theme/images/twitter.png" class="mr-1 twitter-icon">
-                            <b>@MITREattack</b>
-                        </a>
-                    </div>
-                    <div class="">
-                        <a href="/contact" class="btn btn-primary w-100">
-                            Contact
-                        </a>
-                    </div>
+                    {% if ATTACK_BRANDING %}
+                        <div class="mb-1">
+                            <a href="https://twitter.com/MITREattack" class="btn btn-primary w-100">
+                                <!-- <i class="fa fa-twitter"></i> -->
+                                <img src="/theme/images/twitter.png" class="mr-1 twitter-icon">
+                                <b>@MITREattack</b>
+                            </a>
+                        </div>
+                        <div class="">
+                            <a href="/contact" class="btn btn-primary w-100">
+                                Contact
+                            </a>
+                        </div>
+                    {% else %}
+                        <div class="">
+                            <a href="attack.mitre.org" class="btn btn-primary w-100">
```

Comment:
This url needs `https://` because otherwise it tries to go to `[host]/attack.mitre.org`.